### DISCUSSION TITLE
Is it possible to generate methods' phpdoc comments when they are missing?

### DISCUSSION THREAD

**Discussion by polishExperiment** [PHP-CS-Fixer/PHP-CS-Fixer#6963] [Q&A]

**Comment by dennisdahlc** [comment]
I made my own extended fixer which puts in a TODO comment when missing, just to provide some kind of documentation to satisfy the phpdocfixer, since my team does not have the time currently to document everything all at once ;)

**Comment by tryrushio** [comment]
I'm backing this request it will be quite nice!

**Comment by Wirone** [comment]
Personally I believe that with PHP as a language moving more and more towards strict typing it does not make sense to enforce phpDoc where it's missing. There are even fixers for [removing superfluous phpDoc annotations](https://github.com/kubawerlos/php-cs-fixer-custom-fixers#phpdocnosuperfluousparamfixer) or [phpDoc comments](https://github.com/kubawerlos/php-cs-fixer-custom-fixers#nouselesscommentfixer) and this is the direction which we should encourage. There are tools like [PHPStan](https://github.com/phpstan/phpstan) which can help with catching those missing types.

Having said that, we probably won't support such fixer, even if it was contributed by community. But that's only my opinion, last work goes to @keradus 😉.
